common/types/message/message.go

```diff
@@ -1,28 +1,11 @@
 package message
 
 import (
-	"crypto/ecdsa"
-	"crypto/rand"
-	"encoding/hex"
 	"errors"
 	"fmt"
 
+	"github.com/scroll-tech/da-codec/encoding/codecv3"
 	"github.com/scroll-tech/go-ethereum/common"
-	"github.com/scroll-tech/go-ethereum/common/hexutil"
-	"github.com/scroll-tech/go-ethereum/crypto"
-	"github.com/scroll-tech/go-ethereum/rlp"
-)
-
-// ProofFailureType the proof failure type
-type ProofFailureType int
```

Comment:
suggest not removing the definition and enums here.
or we will lose the meaning in this field `FailureType` in `SubmitProofParameter`

Reply:
https://github.com/scroll-tech/scroll/pull/1394/files/796e77d8a31bd96560e3e880da6fb3142e29bfad#diff-6ca81ab4ad75c50e0f3a694a0bcd5563143edb7a7e4935ba9ed2ff949d3084b1R99

failure type is using this one